tests/fsharpqa/Source/Conformance/SpecialAttributesAndTypes/Imported/CallerInfo/CallerMemberName.fs

```diff
@@ -45,6 +47,8 @@ type MyTy() =
             |> List.head
         MyTy.Check(result, Some("CheckMembers"), "lambda")
         MyTy.Check(functionVal (), Some("functionVal"), "functionVal")
+        MyTy.Check(anonymousLambda, Some("anonymousLambda"), "anonymousLambda")
```

Comment:
Although this looks like an anonymous lambda, there is a quirk in the F# language specification where ``let x = fun args -> ...`` is actually considered a normal function definition if the ``let`` is a module binding. I think you should use an example where the ``let`` occurs in expression position.